cmd/achcli/code_map.go

```diff
@@ -0,0 +1,74 @@
+package main
```

Comment:
For the descriptions displayed to the user, I added spaces between them. For example, `Checking Zero Dollar Remittance Credit` over `CheckingZeroDollarRemittanceCredit` -- a bit longer but easier to read.

Any thoughts?

Reply:
Oh nice, I think it looks better with the spaces